setup.py

```diff
@@ -20,6 +20,21 @@ def _get_version():
     return major_minor
 
 
+def _get_data_files():
+    data_files = [
+        ("/etc/apt/apt.conf.d", ["apt.conf.d/51ubuntu-advantage-esm"]),
+        ("/etc/ubuntu-advantage", ["uaclient.conf"]),
+        ("/usr/share/keyrings", glob.glob("keyrings/*")),
+        (defaults.CONFIG_DEFAULTS["data_dir"], []),
+    ]
+    rel_major, _rel_minor = util.get_platform_info()["release"].split(".", 1)
+    if rel_major == "14":
```

Comment:
Does this work as intended if I'm building a trusty source package on a non-trusty system? Or will this select the build system based on the build host?

(This may not be a problem, I just want to understand!)

Reply:
Correct. This will select to package files based on the underlying host platform. Since we removed the ability to cross-platform build in earlier iterations, our commitment to build on the target platform was something  that I relied upon here.

Reply:
We haven't been able to build the _binary_ packages on different hosts for a while, but I believe we can still build the source packages on different hosts. This is a pretty important difference, both for a local sbuild workflow but also for the package builds in Travis.

I'll double-check a source package build with this branch in a couple of minutes.

Reply:
I've just tested this locally and a source build on focal correctly installed the upstart file for trusty and the systemd file for bionic. Marking this resolved.